packages/metal-dom/test/dom.js

```diff
@@ -359,6 +360,73 @@ describe('dom', function() {
 			dom.triggerEvent(element, 'click');
 			assert.strictEqual(0, listener.callCount);
 		});
+
+		it('should listen to keyboard events by keyCode alias', function() {
+			var element = document.createElement('input');
+			element.setAttribute('type', 'text');
```

Comment:
The element you're calling `triggerEvent` on doesn't need to be a text input, so removing this line would make the tests a bit simpler.